src/android/CameraLauncher.java

```diff
@@ -537,17 +540,20 @@ private void processResultFromCamera(int destType, Intent intent) throws IOExcep
         // in the gallery and the modified image is saved in the temporary
         // directory
         if (this.saveToPhotoAlbum) {
-            galleryUri = Uri.fromFile(new File(getPicutresPath()));
+            try {
+                galleryUri = Uri.fromFile(new File(getPicutresPath()));
 
-            if(this.allowEdit && this.croppedUri != null) {
-                writeUncompressedImage(this.croppedUri, galleryUri);
-            } else {
+                if (this.allowEdit && this.croppedUri != null) {
+                    writeUncompressedImage(this.croppedUri, galleryUri);
+                } else {
+                    Uri imageUri = this.imageUri.getFileUri();
+                    writeUncompressedImage(imageUri, galleryUri);
+                }
 
-                 Uri imageUri = this.imageUri.getFileUri();
-                writeUncompressedImage(imageUri, galleryUri);
+                refreshGallery(galleryUri);
+            } catch (IOException e) {
+                TLog.e(TAG, e);
```

Comment:
just in case this plug and fails to save to photo album, catch and log to RayGun, and don't prevent the image from actually being used on Cordova side.